src/ameedf/jb8223/assignments/hw02/Clock.java

```diff
@@ -34,4 +34,127 @@
  * BONUS 2: the same functionality as in "BONUS 1" while using as few member variables / memory as possible
  */
 public class Clock {
+  // Make variables
+	private byte hours;
+	private byte minutes;
+	private byte seconds;
+	private int milliseconds;
+	private boolean twentyFourDisplay;  
```

Comment:
no need for this variable.. it's only used in one method and it come in as a parameter